SConstruct

```diff
@@ -60,14 +64,10 @@ The following environment variables modify the build environment:
     OPENSSL_ROOT
       Path to the openssl directory.
 
-The following extra options may be used:
-    --ninja         Generate a `build.ninja` build file for the specified target
-                    (see: https://martine.github.io/ninja/). Only gcc and clang targets
-                     are supported.
+"""
```

Comment:
We're going to be having at least two SConstructs now - should this really go in both SConstructs?

Honestly, I'd like to get rid of all these comments from here and put them into an actual document.  (Also, I'd like to consider bringing back ninja mode, sorry I dropped that when I was struggling...)